lib/lotus/model/adapters/abstract.rb

```diff
@@ -86,9 +86,10 @@ class Abstract
         # @param uri [String] the optional connection string to the database
         #
         # @since 0.1.0
-        def initialize(mapper, uri = nil)
+        def initialize(mapper, uri = nil, options = {})
```

Comment:
@taylorfinnell Can you please document `options` as `@param` in the API doc? Don't forget the other signatures! :smile: Thank you :+1: